packages/flutter_blue/LICENSE

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright (c) 2017 The Chromium Authors. All rights reserved.
```

Comment:
`flutter_blue` is not written by Chromium Authors.
Copy License file from https://github.com/pauldemarco/flutter_blue/blob/master/LICENSE, then add Samsung copyright notice at the top.